cli/src/command_error.rs

```diff
@@ -487,13 +497,22 @@ fn try_handle_command_result(
             match inner.kind() {
                 clap::error::ErrorKind::DisplayHelp | clap::error::ErrorKind::DisplayVersion => {
                     write!(ui.stdout(), "{clap_str}")?;
-                    Ok(ExitCode::SUCCESS)
+                    return Ok(ExitCode::SUCCESS);
                 }
-                _ => {
-                    write!(ui.stderr(), "{clap_str}")?;
-                    Ok(ExitCode::from(2))
+                _ => {}
+            }
+
+            write!(ui.stderr(), "{clap_str}")?;
+            if let Some(CliArgInvalidValue::Template) = inspect_invalid_value(inner.context()) {
+                match load_template_aliases(ui, layered_configs) {
+                    Ok(aliases_map) => {
+                        let hint = format_symbol_names_hint(&aliases_map);
+                        write!(ui.hint(), "\n{hint}")?;
+                    }
+                    Err(err) => return try_handle_command_result(ui, layered_configs, Err(err)),
```

Comment:
Probably better to suppress the `load_template_aliases()` error. The error here is less important than the original error.